schema/Organization.schema.yaml

```diff
@@ -27,6 +27,15 @@ properties:
     type: array
     items:
       $ref: ContactPoint
+  members:
+    '@id': schema:members
```

Comment:
This should be the `id` of the schema.org property which is `member`, not `members`.